src/clj_discord_bot/core.clj

```diff
@@ -35,23 +36,18 @@
   (let [message (get data "content")]
     (try
       (cond
-        (.contains message "clojure")
-        (evangelize/get-propaganda type data)
-        (.equals "!help" message)
-        (help type data)
-        (.equals "!d20" message)
-        (roll/d20 type data)
-        (.startsWith message "!summon ")
-        (summon/game-summon type data)
-        (.startsWith message "!img ")
-        (img-search/find-img type data)
-        (re-find #"(?i)ghandi" message)
-        (misc/gandhi-spellcheck type data)
-        (re-find #"(?i)link" message)
-        (misc/links-mentioned type data))
+        (.contains message "clojure") (when (common/random-chance 10)
```

Comment:
I think it makes more sense for the random chance logic to live inside of that command instead of here